x11-packages/nextcloudcmd/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/nextcloud/desktop
```

Comment:
This package seems to usually be packaged as `nextcloud-client`,
please rename the package directory accordingly.
https://repology.org/project/nextcloud-client/information

The website should probably also be,
```suggestion
TERMUX_PKG_HOMEPAGE=https://nextcloud.com/
```
instead of the repository link.